components/brave_wallet_ui/stories/locale.ts

```diff
@@ -139,7 +139,7 @@ provideStrings({
   braveWalletButtonContinue: 'Continue',
   braveWalletContinueWithXItems: 'Continue with $1 $2',
   braveWalletButtonNext: 'Next',
-  braveWalletButtonGotIt: 'Got it',
+  braveWalletButtonGotIt: 'View my recovery phrase',
```

Comment:
let's remove the old translation and add one with a new name. "got it" doesn't make sense for the translation var name anymore.